tests/test_app.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 
-from server.app import answer_binary_question
+from app import answer_binary_question
```

Comment:
If you put `PYTHONPATH=/app` into Dockerfile (the later stage), then keeping it in server directory should work.

Reply:
Yes, that would work - but since the project is so simple, I'd rather keep the file tree as is.